### PR TITLE
update README to reflect npm support

### PR DESCRIPTION
**Pull request by koenpunt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,12 +11,24 @@ http://harvesthq.github.io/chosen/
 For **downloads**, see:
 https://github.com/harvesthq/chosen/releases/
 
-### Bower Installation
+### Package managers
 
-Chosen is available for Bower installation:
-`bower install chosen`
+Chosen is available through [Bower](https://bower.io/) and [npm](https://www.npmjs.com),
+_however, the package names are not the same_
```
Comment (tjschuck) [inline README.md]: Missing a period at the end of the sentence.

File: README.md [hunk 2]
```diff
@@ -11,12 +11,24 @@ http://harvesthq.github.io/chosen/
...
+bower install chosen
+```
+
+To install with npm:
+
+```
+npm install chosen-js
+```
+
+The compiled files for these packages are automatically generated and stored in a [2nd Chosen repository](https://github.com/harvesthq/chosen-package). No pull requests will be accepted to that repository.
```
Comment (stof) [inline README.md]: Does npm actually rely on this repo ? I thought it relied on archives uploaded to the registry

**Comment by tjschuck** [comment]
Should we add information about this to https://harvesthq.github.io/chosen/ as well?  Either in Setup (on 1, "...or install via [Bower or npm][link-to-readme-section]"), or in the FAQs ("Is Chosen available on Bower or npm?")?

File: README.md (hunk 2 above)
Comment (koenpunt) [inline README.md]: No it doesn't, but the archive being uploaded is based on the contents of that repo by using the deploy option provided by Travis.

**Pull request merged**